content.js

```diff
@@ -211,20 +206,24 @@ function resetValues() {
     //reset sponsor times
     sponsorTimes = null;
     UUIDs = null;
-    sponsorVideoID = id;
     sponsorLookupRetries = 0;
 
     //empty the preview bar
-    previewBar.set([], [], 0);
+    previewBar = null;
 
     //reset sponsor data found check
     sponsorDataFound = false;
 }
 
 function videoIDChange(id) {
-    //not a url change
-    if (sponsorVideoID == id) return;
-
+    // ID has not changed return
+    if (sponsorVideoID === id) return
+	// Global ID
```

Comment:
Could you fix the tabbing here? 4 spaces

Reply:
yes